<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametryzatorPrintSettingsWdgt</class>
 <widget class="QWidget" name="ParametryzatorPrintSettingsWdgt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pbClose">
        <property name="text">
         <string>Close</string>
        </property>
        <property name="icon">
         <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
          <normaloff>:/katynko/svg/window-close.svg</normaloff>:/katynko/svg/window-close.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>995</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QListWidget" name="listWidget">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="alternatingRowColors">
       <bool>true</bool>
      </property>
      <item>
       <property name="text">
        <string>Regular</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Print Profiles</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Registered keys</string>
       </property>
      </item>
     </widget>
     <widget class="QStackedWidget" name="swSettingsPages">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>7</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>822</width>
             <height>610</height>
            </rect>
           </property>
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0" colspan="3">
             <layout class="QHBoxLayout" name="horizontalLayout_21">
              <item>
               <widget class="QPushButton" name="pbSaveAsDef">
                <property name="text">
                 <string>Save</string>
                </property>
                <property name="icon">
                 <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
                  <normaloff>:/katynko/svg/document-save.svg</normaloff>:/katynko/svg/document-save.svg</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pbLoadSaved">
                <property name="text">
                 <string>Reload</string>
                </property>
                <property name="icon">
                 <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
                  <normaloff>:/katynko/svg/view-refresh.svg</normaloff>:/katynko/svg/view-refresh.svg</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeType">
                 <enum>QSizePolicy::Ignored</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="pbDefault">
                <property name="text">
                 <string>Default</string>
                </property>
                <property name="icon">
                 <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
                  <normaloff>:/katynko/svg/edit-undo.svg</normaloff>:/katynko/svg/edit-undo.svg</iconset>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_17">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_31">
              <property name="text">
               <string>Address</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPlainTextEdit" name="pteAddrs">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="plainText">
               <string>Україна, м. Київ
вул. Бориспільська, 7
тел. +38-068-358-91-49
info@kts-intek.com
www.kts-intek.com.ua
www.kts-intek.com</string>
              </property>
              <property name="placeholderText">
               <string>адреса підприємства</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_22">
              <item>
               <widget class="QToolButton" name="tbPath2pdfDir">
                <property name="text">
                 <string>...</string>
                </property>
                <property name="icon">
                 <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
                  <normaloff>:/katynko/svg/document-open.svg</normaloff>:/katynko/svg/document-open.svg</iconset>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lePath2pdfDir">
                <property name="readOnly">
                 <bool>true</bool>
                </property>
                <property name="placeholderText">
                 <string>path to working directory</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_36">
              <property name="text">
               <string>PDF file name</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lePdfFileName">
              <property name="placeholderText">
               <string>directory/filename</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="leCutAddrs2">
              <property name="placeholderText">
               <string>regular expression</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_35">
              <property name="text">
               <string>Meter version to Meter type</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="QPlainTextEdit" name="pteMeterProg2meterType">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="plainText">
               <string>MTX1A MTX 1A10.DF.2LO-CO4
MTX3R MTX 3R30.Dh.4L1-OG4
CE303 CE303
CE102 CE102
</string>
              </property>
             </widget>
            </item>
            <item row="6" column="0">
             <widget class="QLabel" name="label_32">
              <property name="text">
               <string>Meter version to Voltage Ampers Manufacturer City</string>
              </property>
             </widget>
            </item>
            <item row="6" column="1">
             <widget class="QPlainTextEdit" name="pteMeterModel2VoltageManufCity">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
            <item row="7" column="0">
             <widget class="QLabel" name="label_34">
              <property name="text">
               <string>NI Rule</string>
              </property>
             </widget>
            </item>
            <item row="7" column="1">
             <widget class="QLineEdit" name="leFindNIRegExp">
              <property name="placeholderText">
               <string>regular expression</string>
              </property>
              <property name="clearButtonEnabled">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="8" column="0">
             <widget class="QLabel" name="label_33">
              <property name="text">
               <string>NI changing rule (for files only)</string>
              </property>
             </widget>
            </item>
            <item row="8" column="1">
             <layout class="QHBoxLayout" name="horizontalLayout_23">
              <item>
               <widget class="QLineEdit" name="leReplaceNIrule4fileFind">
                <property name="placeholderText">
                 <string>space is a separator</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="leReplaceNIrule4fileReplace">
                <property name="placeholderText">
                 <string>space is a separator</string>
                </property>
                <property name="clearButtonEnabled">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="9" column="1">
             <spacer name="verticalSpacer_6">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Working directory</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Remove waste from Consumer address</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_24">
          <item>
           <widget class="QPushButton" name="pbSaveHtml">
            <property name="text">
             <string>Save</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
              <normaloff>:/katynko/svg/document-save.svg</normaloff>:/katynko/svg/document-save.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pbLoadHtml">
            <property name="text">
             <string>Reload</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
              <normaloff>:/katynko/svg/view-refresh.svg</normaloff>:/katynko/svg/view-refresh.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Ignored</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pbLoadDefHtml">
            <property name="text">
             <string>Apply default to active profile</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
              <normaloff>:/katynko/svg/edit-undo.svg</normaloff>:/katynko/svg/edit-undo.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_18">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_25">
          <item>
           <widget class="QComboBox" name="cbxPrintProfiles">
            <property name="sizeAdjustPolicy">
             <enum>QComboBox::AdjustToContents</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbAddProfile4print">
            <property name="text">
             <string>+</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
              <normaloff>:/katynko/svg/draw-cross.svg</normaloff>:/katynko/svg/draw-cross.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbRemoveProfile4print">
            <property name="text">
             <string>-</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
              <normaloff>:/katynko/svg3/user-trash-full.svg</normaloff>:/katynko/svg3/user-trash-full.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Ignored</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_37">
            <property name="text">
             <string>Font size by default</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="sbFontSizeHtml">
            <property name="minimum">
             <number>3</number>
            </property>
            <property name="maximum">
             <number>9999</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Ignored</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="tbReplaceKeysInTxtBrwsr">
            <property name="text">
             <string>...</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
              <normaloff>:/katynko/svg2/view-readermode.svg</normaloff>:/katynko/svg2/view-readermode.svg</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbShowTxtBrwsrPrev">
            <property name="text">
             <string>перегляд</string>
            </property>
            <property name="icon">
             <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
              <normaloff>:/katynko/svg2/view-web-browser-dom-tree.svg</normaloff>:/katynko/svg2/view-web-browser-dom-tree.svg</iconset>
            </property>
            <property name="checkable">
             <bool>true</bool>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_5">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeType">
             <enum>QSizePolicy::Ignored</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QToolButton" name="tbPrintPreview">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
              <normaloff>:/katynko/svg/document-print.svg</normaloff>:/katynko/svg/document-print.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbPrintPdf">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../../../../Matilda-units/gui/image-and-font/res.qrc">
              <normaloff>:/katynko/svg/application-pdf.svg</normaloff>:/katynko/svg/application-pdf.svg</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_19">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter_2">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="childrenCollapsible">
           <bool>false</bool>
          </property>
          <widget class="QStackedWidget" name="sw4htmlEditor">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentIndex">
            <number>-1</number>
           </property>
          </widget>
          <widget class="QScrollArea" name="scrollArea_4">
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents_4">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>818</width>
              <height>546</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_29">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QTextBrowser" name="txtBrwsr4prew"/>
             </item>
            </layout>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../../../Matilda-units/gui/image-and-font/res.qrc"/>
 </resources>
 <connections/>
</ui>
